src/react/components/pages/predict/predictPage.tsx

```diff
@@ -838,9 +850,19 @@ export default class PredictPage extends React.Component<IPredictPageProps, IPre
         this.imageMap.addFeatures(features);
     }
 
+    private saveLastPredictionResultToLocalStorage = (result, projectID) => {
```

Comment:
what about only save only one project (last run) result? Don't want to consume too much local storage to exceed to local storage quota

Reply:
Yes, It's make sense since Local Storage is pretty limited.

Reply:
implemented in - d962fd0